.github/workflows/validate-frontend.yaml

```diff
@@ -0,0 +1,24 @@
+name: validate
+on: pull_request
```

Comment:
> on: pull_request

We wouldn't want the job to run for when there are no frontend changes right now.

```
on:
  pull_request:
    paths:
      - 'frontend/**'
```

You can refer to this [build.yaml](https://github.com/SigNoz/signoz/blob/develop/.github/workflows/build.yaml) for the running specific job based on paths.

Reply:
First we tried modifying this only but later we went on creating a new pipeline becuase 👇🏼 
The goal of this (`validate`) job is to only validate the code on the PRs by running lint check and build check, soon we will be extending this to other set of automated code review and validation plugins.